.travis.yml

```diff
@@ -64,6 +64,7 @@ branches:
     - "6.3"
     - "6.4"
     - "6.5"
+    - fix-urlalias-downgrade
```

Comment:
Removed in https://github.com/ezsystems/ezpublish-kernel/pull/1781/commits/48787249cf9d76bef8e62beb1323af4af1a74f2a.